dsn_test.go

```diff
@@ -60,6 +61,7 @@ func TestParseDSN(t *testing.T) {
 				JWTClientTimeout:          defaultJWTClientTimeout,
 				ExternalBrowserTimeout:    defaultExternalBrowserTimeout,
 				IncludeRetryReason:        ConfigBoolTrue,
+				DisableConsoleLogin:       ConfigBoolTrue,
```

Comment:
Do we need to specify it in all tests?

Reply:
Updated. No longer need to specify it in all tests.